tests/processor/test_tasks.py

```diff
@@ -276,6 +276,27 @@ def test_process_file_missing_manifest(self, mock_manifest_accessor, mock_stats_
         mock_manifest_acc.mark_manifest_as_updated.assert_not_called()
         shutil.rmtree(report_dir)
 
+    @patch('masu.processor.tasks.update_summary_tables')
+    def test_summarize_reports(self, mock_update_summary):
```

Comment:
This looks to me like two distinct unit tests. It's not critical, but it would be nice to break this test up into two tests, one for the `assert_not_called()` case and one for the `assert_called()` case.

Second, let's try to make the docstring more useful by describing the expectations we have for the test. For example: `Test that the summarize_reports task is called when <some condition>.` That will make it a lot easier to know what the author thinks the test should be doing, then match that with what the code says.